.changeset/pink-horses-cough.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-permission-react': patch
+'@backstage/plugin-search-backend': patch
+'@backstage/test-utils': patch
+---
+
+Use `createPermission` helper to create valid `Permission` objects in test suites.
```

Comment:
No need to include a changeset for changes to tests, they don't affect the published package